### PR TITLE
Turn repository eligible for Hacktoberfest

### LINKED ISSUE #114: Turn repository eligible for Hacktoberfest
https://hacktoberfest.com/participation/ 

- Add the “hacktoberfest” topic to your repository to opt-in to Hacktoberfest and indicate you’re looking for contributions.
- Apply the “hacktoberfest” label to issues you want contributors to help with in your GitHub or GitLab project.
- Add a CONTRIBUTING.md file with contribution guidelines to your repository.
- Choose issues that have a well-defined scope and are self-contained.
- Adopt a code of conduct to create a greater sense of inclusion and community for contributors.
- Be ready to review pull/merge requests, accepting those that are valid by merging them, leaving an overall approving review, or by adding the “hacktoberfest-accepted” label.
- Reject any spammy requests you receive by labeling them as “spam,” and any other invalid contributions by closing them or labeling them as “invalid.”

### PR DESCRIPTION
**Pull request by leonardobrito**
Fix #114 

- Add Contributing file;
- Add contributing section to readme.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# How to contribute
+
+I'm really glad you're reading this, because we need volunteer developers to help this project.
+
+## Testing
+
+Our tests consists of RSpec examples. Please write RSpec examples for new code you create.
+
+## Submitting changes
+
```
Comment (edimossilva) [inline CONTRIBUTING.md:10]: I think we can add some instruction about the fork
e.g. https://github.com/espoo-dev/espoo-dev/blob/main/CONTRIBUTING.md

**Review by edimossilva — COMMENTED** [review]
When I did it to espoo-dev there was a requirement for adding a https://github.com/espoo-dev/espoo-dev/blob/main/CODE_OF_CONDUCT.md to the repo

File: CONTRIBUTING.md (hunk 1 above)
Comment (leonardobrito) [reply CONTRIBUTING.md:10]: No sure, It may be good if it was mandatory, today is free to decide.

**Comment by leonardobrito** [comment]
> Done 7b19391

File: CONTRIBUTING.md (hunk 1 above)
Comment (leonardobrito) [reply CONTRIBUTING.md:16]: I've added a section to contributors feel free to decide about it.

**Review by edimossilva — APPROVED** [review]
Brilliant

**Pull request merged**